gen/py/chromestatus_openapi/chromestatus_openapi/controllers/default_controller.py

```diff
@@ -0,0 +1,18 @@
+import connexion
+import six
+from typing import Dict
+from typing import Tuple
+from typing import Union
+
+from chromestatus_openapi import util
+
+
+def componentsubscribers_get():  # noqa: E501
+    """componentsubscribers_get
+
+     # noqa: E501
+
+
+    :rtype: Union[None, Tuple[None, int], Tuple[None, int, Dict[str, str]]
+    """
+    return 'do some magic!'
```

Comment:
Where does this come from? 
In other words, where do we write our API handlers?

Reply:
Ah.  I'm thinking that this controller is not used.  The 'do some magic' string is just a placeholder in the generated code and we do not intend to fill it in because we have our own way of setting up python request handlers in main.py.